src/ansys/fluent/core/launcher/standalone_launcher.py

```diff
@@ -185,7 +186,11 @@ def __init__(
         """
         import ansys.fluent.core as pyfluent
 
-        self.argvals, self.new_session = _get_argvals_and_session(locals().copy())
```

Comment:
`pyfluent` was getting copied here which causes `launch_fluent` error in the background session.